set.mm

```diff
@@ -206874,14 +206907,14 @@ that F(A') is isomorphic to B'.".  Therefore, the category of sets and
       UIUQDVACSUJUKUSERZVRWAVHWCVPVTVQWCVOVIVBVSUSEDCTUSEVACSUKULUMVAFRZWAVMVHW
       DVTVKVQVLWDVSVJVIVAFECTUMVAFDCTUJUMUNUO $.
 
-    $( Less-or-equal is reflexive in a proset.  (Contributed by Stefan O'Rear,
-       1-Feb-2015.) $)
+    $( Less-than-or-equal-to is reflexive in a proset.  (Contributed by Stefan
```

Comment:
Why not "Less than or equal to" as above?

Reply:
I don't know ! I found both ways in set.mm and wasn't sure which was best and if I was a bit too "dictatorial" in enforcing one or the other. What's important is that I fixed the few "less or equal to..." to "less than or equal to". If we want to enforce a consistent spelling, we can do it in a future PR.